website/docs/community.mdx

```diff
@@ -0,0 +1,41 @@
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+
+# Community
+Explore the many ways to connect, learn, and participate in the ComposeDB community.
+
+## Chat and Discussion
+- **Forum** - Visit the [Ceramic Forum](https://forum.ceramic.network) to get help, ask questions, and discuss improvements.
+- **Discord** - Join the [Ceramic Discord](https://chat.ceramic.network) to chat in the [#ComposeDB](https://discord.gg/6FCS44YDBs) channel.
+- **Bounties** - Get paid for learning & building on ComposeDB, in the [#bounties](https://discord.com/channels/682786569857662976/1040706471689732096/1040719165268426924) channel
+
+## Social Media
+- **Twitter** - Follow us on Twitter at [@ComposeDB](https://twitter.com/composedb) for timely updates.
+- **YouTube** - Subscribe to the [Ceramic YouTube](https://link.co) to watch talks, tutorials, events, and more.
+
+## Resources
+- **Blog** - Read the [Ceramic Blog](https://blog.ceramic.network) to discover updates and educational content for developers.
+- **Newsletter** - Subscribe to the [Ceramic Newsletter](https://blog.ceramic.network/members/#subscribe) to receive important announcements.
+
+## Events
+- **Community Calls**
+- **Calendar**
+
+## Ways to Contribute
+### Ecosystem Grants
+If you’re interested in a grant feel free to reach out in the #bounties channel. We welcome all submissions, but we’re especially looking for contributions in the following areas:
+
+- **Marketplace GUI:** Wouldn’t it be great if developers could access the composite marketplace with an app UI instead of a terminal?
+- **Sample Apps & Models:** Inspire other developers by building sample apps & data models, like reputation credentials, social apps, or DAO tools.
+- **Easy Node Setup:** Make it easy to deploy ComposeDB on a node - we wrote a [proof of concept](https://github.com/ceramicstudio/ceramic-infra-poc) you can build on. Bonus: Terraform templates for cloud providers like AWS, GCP, and DigitalOcean.
+- **Config Script:** We’d love to make [Set up your environment](./set-up-your-environment) even easier. Want to create a lightning fast script?
+
+### **Open source contributions**
+Contribute to the [ComposeDB codebase](https://github.com/ceramicstudio/js-composedb): here are some good repositories to get started
+
+- [composedb/client](https://github.com/ceramicstudio/js-composedb/tree/main/packages/client)
+- [composedb/cli](https://github.com/ceramicstudio/js-composedb/tree/main/packages/cli)
+
+### **Project showcase**
+- Reach out on the Ceramic Discord channel [#ComposeDB](https://discord.gg/6FCS44YDBs) and tag `@core-devs` to show off your project on CompsoeDB
```

Comment:
Can we avoid asking people to explicitly tag us in Discord please? It's more noise than anything.